en/guide/index.md

```diff
@@ -22,57 +22,57 @@ As a framework, Nuxt.js comes with a lot of features to help you in your develop
 
 ## How it Works
 
-![Vue with Webpack and Babel](https://i.imgur.com/avEUftE.png)
+![Vue with webpack and Babel](https://i.imgur.com/avEUftE.png)
 
 Nuxt.js includes the following to create a rich web application development:
 
 - [Vue 2](https://github.com/vuejs/vue)
-- [Vue-Router](https://github.com/vuejs/vue-router)
+- [Vue Router](https://github.com/vuejs/vue-router)
 - [Vuex](https://github.com/vuejs/vuex) (included only when using the [store option](/guide/vuex-store))
-- [Vue-Meta](https://github.com/declandewet/vue-meta)
+- [Vue Meta](https://github.com/declandewet/vue-meta)
```

Comment:
We see the package name is “vue-meta” and following the official documentation, it seems the product name is “vue-meta” also.
We suggest you to use the same way has official Vue to name your product like “Vue Meta”.

But if you prefer kept your “vue-meta” product name, just replace the original “[Vue Meta](https://github.com/declandewet/vue-meta)” by “[vue-meta](https://github.com/declandewet/vue-meta)”.

What are your thoughts?